### PR TITLE
check connection to registry

### PR DESCRIPTION
**Pull request by AndreySinenok**
GCCP-7816

### CODE REVIEW

File: chartman.sh [hunk 1]
```diff
@@ -121,4 +121,12 @@ resolvedArgs=$(eval echo \"$runDockerArgs\")
 trace "Resolved args: $resolvedArgs"
 trace "docker run --rm $dockerArgs $resolvedArgs -w $PWD $dockerImage:$requestedVersion"
 
+if [ "$runDockerArgs" == "" ]; then
```
Comment (ivanbenko) [inline chartman.sh:124]: better to check `$requestedVersion` since its absence causes the problem

File: chartman.sh [hunk 2]
```diff
@@ -121,4 +121,12 @@ resolvedArgs=$(eval echo \"$runDockerArgs\")
...
+  echo "Checking connection to docker registry"
```
Comment (ivanbenko) [inline chartman.sh:125]: the idea was to capture output of underlying command and output it here, it should contain enough diagnostic info about the problem

Comment (ivanbenko) [reply chartman.sh:125]: take a look at line 57

**Review by ivanbenko — CHANGES_REQUESTED** [review]

File: chartman.sh (hunk 2 above)
Comment (ivanbenko) [reply chartman.sh]: @AndreySinenok this is not done

File: chartman.sh [hunk 3]
```diff
@@ -121,4 +121,18 @@ resolvedArgs=$(eval echo \"$runDockerArgs\")
 trace "Resolved args: $resolvedArgs"
 trace "docker run --rm $dockerArgs $resolvedArgs -w $PWD $dockerImage:$requestedVersion"
 
+if [ "$requestedVersion" == "" ]; then
+  ping -c 1 docker-registry.ds.mhie.com;
```
Comment (alekseysavostin) [inline chartman.sh:125]: take registry from variable

**Pull request merged**